docker/Dockerfile

```diff
@@ -68,7 +68,15 @@ RUN apt-get update -y && \
     apt-get -qy clean && \
     rm -rf /var/lib/apt/lists/* /tmp/osie/
 
-ARG PACKET_HARDWARE_COMMIT=68a25da2b6a3d3281838bff6b62639438116d0d1
+# ironlib cli wrapper, a prereq for packet-hardware since we don't run it from its container
+COPY lfs/getbiosconfig /tmp/osie/
+RUN cd /tmp/osie && \
+    if [ $(uname -m) = 'x86_64' ]; then \
```

Comment:
Always quote strings containing variables, command substitutions, spaces or shell meta characters, unless careful unquoted expansion is required or it’s a shell-internal integer 

Stolen from https://google.github.io/styleguide/shellguide.html - but shellcheck also finds this: https://github.com/koalaman/shellcheck/wiki/SC2046

Reply:
Agreed. You're catching things that are an issue throughout the Dockerfile, so I've updated them everywhere for consistency.